code/controllers/subsystem/changelog.dm

```diff
@@ -91,17 +94,15 @@ SUBSYSTEM_DEF(changelog)
 			winset(C, "rpane.changelog", "background-color=none;text-color=#000000")
 		return
 
-	// Sanity check to ensure clients still exist (If a client DCs mid startup this would runtime)
-	if(C && C.prefs)
-		// If we are ready, process the button style
-		if(C.prefs.lastchangelog != current_cl_timestamp)
-			winset(C, "rpane.changelog", "background-color=#bb7700;text-color=#FFFFFF;font-style=bold")
-			to_chat(C, "<span class='info'>Changelog has changed since your last visit.</span>")
+	// If we are ready, process the button style
+	if(C.prefs.lastchangelog != current_cl_timestamp)
+		winset(C, "rpane.changelog", "background-color=#bb7700;text-color=#FFFFFF;font-style=bold")
+		to_chat(C, "<span class='info'>Changelog has changed since your last visit.</span>")
+	else
+		if(C.prefs.toggles & PREFTOGGLE_UI_DARKMODE)
+			winset(C, "rpane.changelog", "background-color=#40628a;text-color=#FFFFFF")
 		else
```

Comment:
This could also be done with a var right? I can't suggest due to the deletes in the middle

Reply:
??

Reply:
Ah I meant something like:
```dm
var/bg_color
var/text_color
if(C.prefs.toggles & PREFTOGGLE_UI_DARKMODE)
 bg_color = "#40628a"
 text_color = "#FFFFFF"
else
 bg_color = "none"
 text_color = "#000000"
winset(C, "rpane.changelog", "background-color=[bg_color];text-color=[text_color]")
```
Though I initially missed the fact that the background color could also differ. So this method wouldn't really make it better with 2 vars being different.